src/allencell_ml_segmenter/prediction/model.py

```diff
@@ -4,32 +4,16 @@
 
 from allencell_ml_segmenter.core.event import Event
 from allencell_ml_segmenter.core.publisher import Publisher
+from allencell_ml_segmenter.core.file_input_model import FileInputModel
 
 
-class PredictionInputMode(Enum):
-    FROM_PATH = "from_path"
-    FROM_NAPARI_LAYERS = "from_napari_layers"
-
-
-class PredictionModel(Publisher):
+class PredictionModel(FileInputModel):
     """
     Stores state relevant to prediction processes.
     """
 
     def __init__(self) -> None:
         super().__init__()
-
-        # state related to PredictionFileInput
-        self.config_name: Optional[str] = None
-        self.config_dir: Optional[Path] = None
-        self._input_image_path: Optional[Path] = None
-        self._image_input_channel_index: Optional[int] = None
```

Comment:
This all went into `FileInputModel` ?

Reply:
config_name and config_dir are dead code.

input_image_path and channel index are used in both thresholding and prediction in `FIleInput` so they have been moved to `FileInputModel`